store/messagestore_sqlite.go

```diff
@@ -47,6 +80,31 @@ func (p *persistentMessageStore) Get(id state.MessageID) (*protobuf.Message, err
 	); err != nil {
 		return nil, err
 	}
+
+	message.Metadata = &protobuf.Metadata{Ephemeral: false, Parents: make([][]byte, 0)}
```

Comment:
```suggestion
	message.Metadata = &protobuf.Metadata{Ephemeral: false}
```
is the same but does not allocate memory for `Parents` by default. Go can handle `append` on `nil`.